src/UXClient/Components/HierarchyNavigation/HierarchyNavigation.ts

```diff
@@ -446,7 +446,8 @@ class HierarchyNavigation extends Component{
      // do exact search with tsid to retrieve all possible paths until that instance to traverse for expansion
      private doExactSearchWithPossiblePaths = (tsid, hNames) => {
         this.setModeAndRequestParamsForFilter();
-        this.searchString = '"' + tsid.join(" ").replace(":", " ") + '"'; //TODO: null vs string null check for exact search and escape for character : fix from backend will come here!!
+        let escapedTsidString = tsid.join(" ").replaceAll(/["!\(\)\^\[\{:\]\}~/ @#\$%\&\*;'\+=._-]/g, '+'); //escaping non alphanumeric characters with + for exact search with quotes
```

Comment:
this comment is no longer accurate.  if we are in fact just replacing a handful of special characters with "+", i think its best if the code actually does that naively (using an array of the characters, a foreach, and a string replace) instead of a regex.  it will make it more maintainable for other people, and perf isnt really a concern here